### ISSUE TITLE
nixfmt incorrectly transforms strings to double single quote form

### ISSUE THREAD

**Issue by florpe** [serokell/nixfmt#96]
### Description

Nixfmt (sometimes?) converts strings to double single quote form. For certain strings, the transformation is incorrect. No transformation at all would be preferrable.

### Small example input
```
  someValue = ",\n    ";
```
### Expected output
```
  someValue = ",\n    "; # Unchanged, no transformation necessary
```
### Actual output
```
  someValue = ''
    ,
        ''; # Equivalent to "\n,"
```

**Comment by parenthetical** [comment]
Another case:
### Sample input
```
"'\${x:y}'\n"
```
### Expected output
```
''
  ''\'''${x:y}'
''
```
### Actual output
```
''
  '''${x:y}'
''
```

**Comment by yorickvP** [comment]
Yep! Fixed in v0.5.0!
